app/components/channel_modal.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+import {Animated, Dimensions} from 'react-native';
+
+const AnimatedView = Animated.View;
+const {width: deviceWidth, height: deviceHeight} = Dimensions.get('window');
+
+export default class ChannelModel extends Component {
+    static propTypes = {
+        children: PropTypes.node,
+        duration: PropTypes.number,
+        topOffset: PropTypes.number,
+        theme: React.PropTypes.object.isRequired,
+        visible: PropTypes.bool.isRequired
+    }
+
+    static defaultProps = {
+        duration: 400,
```

Comment:
Eslint would consider this a magic number and warn us about it because of the `no-magic-numbers` rule.
For numeric values in inline/embedded styles, I have mixed feelings about the applicability of this rule (see comment below), but this animation duration here is a good example of a reusable value that could be extracted into a set of constants.
For examples of extractable values, see [constants.jsx](https://github.com/mattermost/platform/blob/master/webapp/utils/constants.jsx) in the webapp.